### ISSUE TITLE
[FEATURE] Subscribe to Only Channel Videos, Shorts, and/or Streams

### ISSUE THREAD

**Issue by mohamed-sallam** [TeamNewPipe/NewPipe#11079]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

Add checkboxes to a list that appears when the "Subscribe" button is clicked (or equivalent technique). This list allows users to choose what they want to subscribe to on the channel: all content, videos, shorts, and/or live streams.

### Why do you want this feature?

Users can subscribe to a channel without cluttering their "What's New" page with unwanted shorts or live streams.

**Comment by mohamed-sallam** [comment]
@SameenAhnaf I think so

**Comment by natewind** [comment]
> Wouldn't #10676 solve this?

I don’t think it would, not completely. What if you want a flat list of videos, without the complication of different tabs, but you watch only shorts from some channels and only non-shorts from others?
